pkg/network/encoding/format.go

```diff
@@ -42,6 +42,8 @@ func FormatConnection(conn network.ConnectionStats, domainSet map[string]int, ro
 	c.PidCreateTime = 0
 	c.LastBytesSent = conn.LastSentBytes
 	c.LastBytesReceived = conn.LastRecvBytes
+	c.LastPacketsSent = conn.LastSentSegs
```

Comment:
Let's be consistent with the naming. This appears to be only place we call this statistic "packets", so perhaps we should change it to `LastSegmentsSent` and `LastSegmentsReceived`?